src/resources/language/_language-zh.yml

```
@@ -51,3 +51,22 @@ crossref-eq-prefix: "方程式"
 crossref-lof-title: "插图目录"
 crossref-lot-title: "列表目录"
 crossref-lol-title: "列表目录"
+
+listing-page-order-by: "排序方式"
+listing-page-order-by-default: "缺省"
+listing-page-order-by-date-asc: "最老"
+listing-page-order-by-date-desc: "最新"
+listing-page-order-by-number-desc: "降序"
```

assuming "number" means page number here then the translation would be this:

```suggestion
listing-page-order-by-number-desc: "页码降序"
```

---

This one is a little confusing - these don't representing ordering by page numbers, but instead more generally ordering by a numeric value. 

So for example, we currently say

'High to Low' in the en translation, but it could also be 'Largest to smallest' or 'Descending'. I very much appreciate your help and would love a recommendation given this new context. Thank you!!

---

OK. Then I think the original version is fine.

---

Thank you very much with your help with this- we really appreciate it!